mantle/cmd/ore/aws/upload.go

```diff
@@ -182,7 +182,9 @@ func runUpload(cmd *cobra.Command, args []string) error {
 	s3ObjectPath := strings.TrimPrefix(s3URL.Path, "/")
 
 	if uploadForce {
-		API.RemoveImage(uploadAMIName, s3BucketName, s3ObjectPath)
+		if err := API.RemoveImage(uploadAMIName, s3BucketName, s3ObjectPath); err != nil {
+			return err
```

Comment:
Let's match the error-handling used in the rest of this function, i.e. `fmt.Fprintf()` and `os.Exit()` ?

Reply:
Thank you for review. Fixed.